common/ledger/blkstorage/blockindex.go

```diff
@@ -267,6 +305,28 @@ func constructTxIDKey(txID string, blkNum, txNum uint64) []byte {
 	return append(k, util.EncodeOrderPreservingVarUint64(txNum)...)
 }
 
+// decodeTxIDKey takes input an encoded txid key of the format `prefix:len(TxID):TxID:BlkNum:TxNum`
+// and returns the bytes segment `len(TxID):TxID` from this
+func decodeTxIDKey(encodedTxIDKey []byte) ([]byte, error) {
+	if len(encodedTxIDKey) == 0 {
+		return nil, errors.New("Invalid txIDKey - zero-length slice")
+	}
+	if encodedTxIDKey[0] != txIDIdxKeyPrefix {
+		return nil, errors.Errorf("Invalid txIDKey {%x} - unexpected prefix", encodedTxIDKey)
+	}
+	remainingBytes := encodedTxIDKey[utf8.RuneLen(txIDIdxKeyPrefix):]
```

Comment:
```suggestion
	keyWithoutPrefix := encodedTxIDKey[utf8.RuneLen(txIDIdxKeyPrefix):]
```
Subjective point of view. It makes it more readable.

Reply:
I'll like to retain this variable name as is for now.